src/payment/strategies/paypal-commerce/paypal-commerce-payment-initialize-options.ts

```diff
@@ -77,18 +77,23 @@ export type PaypalCommerceInitializeOptions = PaypalCommercePaymentInitializeOpt
 export interface PaypalCommercePaymentInitializeOptions {
     /**
      * The CSS selector of a container where the payment widget should be inserted into.
+     * It's necessary to specify this parameter when using Paypal.
+     * Without it Paypal will not work.
+     * For example if you have element with id="paypal-container" - then your container will look like "#paypal-container"
      */
     container: string;
 
     /**
      *  The Client ID of the Paypal App
+     *  You wont be able to initialize Paypal without Client ID
```

Comment:
I'm not sure this is appropriate. Anything that doesn't have a `?:` is required. We could make it more apparent when we generate the documentation, but I don't think we should be adding redundant text all across the codebase.

Reply:
Fixed